policyengine_us/parameters/gov/states/ct/tax/income/rates/head_of_household.yaml

```diff
@@ -11,15 +11,19 @@ metadata:
       href: https://portal.ct.gov/-/media/DRS/Forms/2023/Income/2023-CT-1040-Instructions_1223.pdf#page=20
     - title: 2022 Form CT-1040 Connecticut Resident Income Tax Return Instructions
       href: https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=20
+    - title: Connecticut General Statutes, Chapter 229, Sec. 12-700, 2024 Supplement
```

Comment:
```suggestion
    - title: Connecticut General Statutes, Chapter 229, Sec. 12-700, (10)(A)(i)
```

Reply:
And all other yaml files

When referencing the legal code, we want to be as precise and possible - reference the exact subsection of the law where the parameters are mentioned